lang/en/tool_dataflows.php

```diff
@@ -207,6 +207,8 @@
 $string['path_invalid'] = 'Path "{$a}" is not permitted.';
 $string['path_not_absolute'] = 'Path "{$a}" is not an absolute path.';
 $string['unknown_placeholder'] = 'Placeholder "{$a}" is not recognised.';
+$string['no_dot_installed'] = 'No valid path to Graphviz dot found (Have you installed it?).';
+$string['no_dot_installed_short'] = 'Graphviz not found';
```

Comment:
```suggestion
$string['no_dot_installed_short'] = 'Graphviz not installed';
```
Or perhaps this would be clearer?
```suggestion
$string['no_dot_installed_short'] = 'Graphviz not installed - dot unavailable';
```

Reply:
It could be that the specified path is wrong.

Reply:
Would you be able to make this clearer so the user can tell whether or not they need to install it, or need to fix the path. I think that would be the sweet spot.